addons/hr_attendance/controllers/main.py

```diff
@@ -52,6 +52,7 @@ def _get_geoip_response(mode, latitude=False, longitude=False):
 
     @http.route('/hr_attendance/kiosk_mode_menu', auth='user', type='http')
     def kiosk_menu_item_action(self):
+        # TODO: remove in master in favor of always using route with company_id suffix
```

Comment:
this comment can go - or we could rewrite as: 
`# better use route with company_id suffix`